{
  "cveId": "CVE-2025-10157",
  "eventName": "New CVE Received",
  "cveChangeId": "6AD60F8F-CFCD-4695-861C-30B41DAFC3E8",
  "sourceIdentifier": "reefs@jfrog.com",
  "created": "2025-09-17T12:15:38.097",
  "details": [
    {
      "action": "Added",
      "type": "Description",
      "newValue": "A Protection Mechanism Failure vulnerability in mmaitre314 picklescan versions up to and including 0.0.30 allows a remote attacker to bypass the unsafe globals check. This is possible because the scanner performs an exact match for module names, allowing malicious payloads to be loaded via submodules of dangerous packages (e.g., 'asyncio.unix_events' instead of 'asyncio'). \n\nWhen the incorrectly considered safe file is loaded after scan, it can lead to the execution of malicious code."
    },
    {
      "action": "Added",
      "type": "CVSS V4.0",
      "newValue": "AV:N/AC:L/AT:N/PR:N/UI:N/VC:H/VI:H/VA:H/SC:N/SI:N/SA:N/E:X/CR:X/IR:X/AR:X/MAV:X/MAC:X/MAT:X/MPR:X/MUI:X/MVC:X/MVI:X/MVA:X/MSC:X/MSI:X/MSA:X/S:X/AU:X/R:X/V:X/RE:X/U:X"
    },
    {
      "action": "Added",
      "type": "CWE",
      "newValue": "CWE-693"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://github.com/mmaitre314/picklescan/blob/2a8383cfeb4158567f9770d86597300c9e508d0f/src/picklescan/scanner.py#L309"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://github.com/mmaitre314/picklescan/security/advisories/GHSA-f7qq-56ww-84cr"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://huggingface.co/iluem/linux_pkl/resolve/main/asyncio_asyncio_unix_events___UnixSubprocessTransport__start.pkl"
    }
  ]
}